### PR TITLE
feature: display tx traces

### PR DESCRIPTION
**Pull request by r1b**

### CODE REVIEW

File: src/components/TransactionInfo.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ const TransactionInfo = ({ info, receipt }) => (
       <dd>{info.input}</dd>
     </dl>
     {receipt && receipt.logs.length > 0 && <TransactionLogs logs={receipt.logs} />}
+    {trace && trace.length > 0 && <TransactionTrace trace={trace} />}
```
Comment (ptsteadman) [inline src/components/TransactionInfo.js]: I think for both of these it'd be nice to say "no trace" or "no logs" when these are 0.

**Review by ptsteadman — APPROVED** [review]
Consider adding prop-types (see my other PR)

**Comment by r1b** [comment]
Will fix these shortly

**Comment by r1b** [comment]
Going to make a pass at master and add proptypes / pure functional components where i missed them

**Pull request merged**